src/pages/projects-page/ProjectForm.tsx

```diff
@@ -119,7 +121,16 @@ export function ProjectForm(props: Props) {
                     {i18n._(
                       'Work results and achievements reflecting your contributions; For instance, your key-results',
                     )}
-                    <Guide guideText={selfReviewProjectReviewHelpModalText} title={i18n._('Accomplishments')} />
+                    {selfReviewProjectReviewHelpModalText ? (
+                      <Guide
+                        guideText={
+                          <ReactMarkdown renderers={defaultRenderers}>
+                            {selfReviewProjectReviewHelpModalText}
+                          </ReactMarkdown>
+                        }
+                        title={i18n._('Accomplishments')}
```

Comment:
"Accomplishments" doesn't have any translation in `messages.po`. I changed it to the already existing "Accomplishments Description" in #391, but I think that will result in a merge conflict.

Reply:
I changed to `Accomplishments Description`.